include/mim/detail/func/func_vector1.inl

```diff
@@ -55,7 +55,7 @@ namespace mim
 	template <typename T, qualifier Q>
 	constexpr T VectorT<1, T, Q>::distance(const VectorT<1, T, Q>& v) const
     {
-        return mim::math::sqrt(x - v.x);
+        return mim::math::sqrt((x - v.x) * (x - v.x));
```

Comment:
This should actually be mim::math::abs(this->x - v.x); to actually calculate the distance between two values.

Reply:
The current approach does work for non-negative numbers but now that abs in mim is constexpr we can actually use it here.

Reply:
Ok ! going to change that